src/r3/atomic-swap/corda/evm-bridge-workflows/src/main/kotlin/com/r3/corda/evmbridge/services/Connection.kt

```diff
@@ -335,6 +340,10 @@ internal class Connection(private val connectionId: ConnectionId) {
         )
     }
 
+    fun mapTransactionReceipt(receipt: org.web3j.protocol.core.methods.response.TransactionReceipt): com.r3.corda.cno.evmbridge.dto.TransactionReceipt {
```

Comment:
Think this function name is confusing as I would assume it applied a mapping function, not returned the serialized form of something

Reply:
it is not serialized, but 'to serializable' so it is a mapping form a non serialiizable to a serializable dto